src/sentry/models/dashboard_widget.py

```diff
@@ -114,6 +118,61 @@ class Meta:
     __repr__ = sane_repr("widget", "type", "name")
 
 
+class DashboardWidgetQueryOnDemandManager(Manager):
+    def filter_enabled(self):
+        return super().get_queryset().filter(extraction_reason__startswith=_ON_DEMAND_ENABLED_KEY)
```

Comment:
Is `extraction_reason__startswith` supposed to be `extraction_state__startswith`?